package.json

```diff
@@ -7,6 +7,7 @@
     "@bentley/context-registry-client": "2.12.1",
     "@bentley/frontend-authorization-client": "2.12.1",
     "@bentley/geometry-core": "2.12.1",
+    "@bentley/hypermodeling-frontend": "2.11.1",
```

Comment:
why not 2.12.1? (master recently updated to 2.12.1)

Reply:
Added that dep a couple weeks ago on this branch. Will update.

Reply:
Updated.